### PR TITLE
feat: Add clear variant to Collapsible component

### PR DESCRIPTION
**Pull request by lloydstubber**
# Objective
- Update the Collapsible component to have a clear variant, that when open, will have a border on the bottom of it. This change is also keeping the default the way it is and tests/snapshots have been updated.

# Motivation and Context
- This has been updated so we can reflect the new designs for the survey overview page. Where a collapsible component can be open with a border bottom on it.
- Closes this: https://cultureamp.atlassian.net/secure/RapidBoard.jspa?rapidView=186&projectKey=EARTH&modal=detail&selectedIssue=EARTH-307&quickFilter=194

# Screenshots
## Designs
Screen Shot 2021-04-15 at 11 14 47 am

## Storybook
Screen Shot 2021-04-15 at 11 15 25 am

# Checklist

- [ ] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

**Comment by lloydstubber** [comment]
I wanted to open this PR to get the code looked and to make sure I'm doing release processes correctly. As for the design implementation, I've done the border 100% width and just wanted to get eyes on this design. Is this correct to make the line 3px shy of the container? I can update it to do so but was unsure this was correct.

Screen Shot 2021-04-15 at 11 19 09 am

**Comment by katsambarcus** [comment]
> I wanted to open this PR to get the code looked and to make sure I'm doing release processes correctly. As for the design implementation, I've done the border 100% width and just wanted to get eyes on this design. Is this correct to make the line 3px shy of the container? I can update it to do so but was unsure this was correct.
> 
> Screen Shot 2021-04-15 at 11 19 09 am

@lloydstubber yep, good pickup... that divider should go all the way to the edge ;-)

**Comment by lloydstubber** [comment]
No worries! Will leave as is @katsambarcus

File: draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.tsx [hunk 1]
```diff
@@ -23,7 +25,7 @@ export type Props = {
   onToggle?: (open: boolean, id: string) => void
 
   /* Highlights the header when open by changing the background color */
```
Comment (talbet) [inline draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.tsx:27]: Can you update this comment to better describe what the new prop does?

Comment (lloydstubber) [reply draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.tsx]: @talbet Done!

**Review by talbet — APPROVED** [review]
I am happy for this to go through. You might need to reach out to foundations to debug the CI pipeline though

**Comment by lloydstubber** [comment]
@talbet Yep! Chased down the error in a Slack thread and @ActuallyACat is now across it.

**Review by talbet — APPROVED** [review]

**Pull request merged**